packages/react/src/tooltip/root/useTooltipRoot.ts

```diff
@@ -128,7 +126,7 @@ export function useTooltipRoot(params: useTooltipRoot.Parameters): useTooltipRoo
     enabled: !disabled,
     mouseOnly: true,
     move: false,
-    handleClose: hoverable && trackCursorAxis !== 'both' ? safePolygon() : null,
+    handleClose: hoverable ? safePolygon() : null,
```

Comment:
Currently unsolved: the root hook needs to know the value of the context provider but it's a child of the Root component

Reply:
Two options:

1. Use a ref to synchronize the `trackCursorAxis` provider value. Requires forking `useHover` to make `handleClose` a callable function so it can read a ref value inside the event handler (added as a new commit)
2. Use state + a re-render to synchronize the `trackCursorAxis` provider value